### DISCUSSION TITLE
authorize.net

### DISCUSSION THREAD

**Discussion by kc9ryt** [pretix/pretix#3081] [Q&A]
Are there plans to develop a payment plugin for authorize.net?

**Comment by raphaelm** [comment]
We have built one because a customer commissioned us to do so, but haven't released it yet, in part because it is still pretty new and causing some problems and in part because we haven't discussed with the customer whether they'd be okay with it.

That said, if you can, choose a different payment provider, the experience with them wasn't really great so far.
